code-challenges.test.js

```diff
@@ -21,30 +20,120 @@ const colors1 = ["purple", "blue", "green", "yellow", "pink"]
 const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+    // PSUEDO CODE
+    // create function that takes an array
+    // removes first item from array
+    // shuffles array
```

Comment:
recommend explaining how you will shuffle the array

Reply:
This shuffle method was cowboy code